network/network.go

```diff
@@ -300,7 +279,7 @@ func (c *Collector) getNeighbours(n int, subsets []v1.EndpointSubset) ([]string,
 	// Calculate n neighbours, given our local IP and all other net-exporter IPs.
 	neighbours := c.calculateNeighbours(n, ip, addresses)
 
-	c.logger.Log("level", "info", "message", "calculated neighbours", "ip", ip, "neighbours", strings.Join(neighbours, ", "), "scrapeID", c.scrapeID)
+	c.logger.Log("level", "info", "message", "calculated neighbours", "ip", ip, "neighbours", strings.Join(neighbours, ", "))
```

Comment:
Unsure if this should be dropped as well?

Reply:
I'd drop it, I feel like I added this for debug

Reply:
I'll leave it for now. It's still a bit spammy, but I have a strong feel that this information will be useful one day when diagnosing problems. We can drop it if it's annoying.